graf3d/eve/src/TEveDigitSet.cxx

```diff
@@ -410,9 +410,9 @@ void TEveDigitSet::DigitSelected(Int_t idx)
 
 void TEveDigitSet::SecSelected(TEveDigitSet* qs, Int_t idx)
 {
-   Long_t args[2];
-   args[0] = (Long_t) qs;
-   args[1] = (Long_t) idx;
+   Longptr_t args[2];
+   args[0] = (Longptr_t) qs;
+   args[1] = (Longptr_t) idx;
 
```

Comment:
Does `TQObject::Emit()` method was adjusted?
According to docu https://root.cern/doc/master/classTQObject.html#a61ac18c7a8216bf941add95d7726268a multiple arguments should be converted to `Long_t`. Did you check this?

Reply:
The `TQxxx` classes were changed a while ago, for example in [this merged PR](https://github.com/root-project/root/pull/7778/files). But you're right, I'll change the documentation of `TQObject`. Thanks @linev !

Reply:
@linev https://github.com/root-project/root/pull/8411